### PR TITLE
Update the table of the specific support

### PR DESCRIPTION
**Pull request by szadam**
Preview: https://szadam.github.io//blog/2022/11/update-on-pmdk-and-our-long-term-support-strategy/

- - -
This change is [Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/329)

### CODE REVIEW

**Review by lukaszstolarczuk — COMMENTED** [review]
> Reviewed all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/329)* status: 0 of 1 files reviewed, 1 unresolved discussion (waiting on @szadam)

___
*[`content/blog/2022/update-pmdk-project-status.md` line 75 at r1](https://reviewable.io/reviews/pmem/pmem.github.io/329#-NNq1OGU-VAmRzlhK0gl:-NNq1OGU-VAmRzlhK0gm:bppb7gd) ([raw file](https://github.com/pmem/pmem.github.io/blob/ae342ef30c36ac1dcedfe816ed156b94421e67c1/content/blog/2022/update-pmdk-project-status.md#L75)):*
> ```Markdown
> | Libpmemkv | https://github.com/pmem/pmemkv | No further support or maintenance planned |
> | Libminiasync | https://github.com/pmem/miniasync | No further support or maintenance planned |
> | pmdk-convert | https://github.com/pmem/pmdk-convert | No further support or maintenance planned |
> ```

soo... should we update convert somehow...? :wink:

**Comment by szadam** [comment]
*[`content/blog/2022/update-pmdk-project-status.md` line 75 at r1](https://reviewable.io/reviews/pmem/pmem.github.io/329#-NNq1OGU-VAmRzlhK0gl:-NNq3yT38YFPDTGS2288:b-ylofgl) ([raw file](https://github.com/pmem/pmem.github.io/blob/ae342ef30c36ac1dcedfe816ed156b94421e67c1/content/blog/2022/update-pmdk-project-status.md#L75)):*
<details><summary><i>Previously, lukaszstolarczuk (Łukasz Stolarczuk) wrote…</i></summary><blockquote>

soo... should we update convert somehow...? :wink: 
</blockquote></details>

I changed it ;)

File: content/blog/2022/update-pmdk-project-status.md [hunk 1]
```diff
@@ -66,15 +66,17 @@ Andy Rudoff – Persistent Memory Software Architect, Intel Labs
...
-| Libpmemstream | https://github.com/pmem/pmemstream | Issue Reporting and Bug Fixes via GitHub and pmdk_support@intel.com<br /> Seeking new co-maintainer(s) to continue development towards production release |
+| pmdk-convert | https://github.com/pmem/pmdk-convert | Issue Reporting and Bug Fixes via GitHub and pmdk_support@intel.com |
+| Libpmemstream | https://github.com/pmem/pmemstream | No further support or maintenance planned |
 | Libpmemblk<br /> Libpmemlog | https://github.com/pmem/pmdk | No further support or maintenance planned |
 | Libvmem | https://github.com/pmem/vmem | No further support or maintenance planned |
 | Libpmemobj-cpp | https://github.com/pmem/libpmemobj-cpp | No further support or maintenance planned |
 | Libpmemkv | https://github.com/pmem/pmemkv | No further support or maintenance planned |
 | Libminiasync | https://github.com/pmem/miniasync | No further support or maintenance planned |
-| pmdk-convert | https://github.com/pmem/pmdk-convert | No further support or maintenance planned |
```
Comment (wlemkows) [inline content/blog/2022/update-pmdk-project-status.md:75]: Please do not change pmdk-convert line.

**Review by grom72 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/329)* status: 0 of 1 files reviewed, 3 unresolved discussions (waiting on @lukaszstolarczuk and @szadam)

___
*[`content/blog/2022/update-pmdk-project-status.md` line 63 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/329#-NNqNAA-0Teg_pmY8Xof:-NNqNAA-0Teg_pmY8Xog:bszcbnj) ([raw file](https://github.com/pmem/pmem.github.io/blob/c5d2966fb68552c372d0c0cbc9beb113bb1c0e36/content/blog/2022/update-pmdk-project-status.md#L63)):*
> ```Markdown
> | Library | URL | Support Summary |
> |---|---|---|
> | libpmem<br /> libpmem2<br /> libpmemobj<br /> libpmempool<br /> pmempool<br /> pmreorder | https://github.com/pmem/pmdk | Issue Reporting and Bug Fixes via GitHub and pmdk_support@intel.com<br /> Release planned for Crow Pass<br /> Seeking new co-maintainer(s) to continue development towards production release |
> ```

Is it still valid?
___
*[`content/blog/2022/update-pmdk-project-status.md` line 64 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/329#-NNqNMFp6VWYPvwCnCBZ:-NNqNMFp6VWYPvwCnCB_:bszcbnj) ([raw file](https://github.com/pmem/pmem.github.io/blob/c5d2966fb68552c372d0c0cbc9beb113bb1c0e36/content/blog/2022/update-pmdk-project-status.md#L64)):*
> ```Markdown
> |---|---|---|
> | libpmem<br /> libpmem2<br /> libpmemobj<br /> libpmempool<br /> pmempool<br /> pmreorder | https://github.com/pmem/pmdk | Issue Reporting and Bug Fixes via GitHub and pmdk_support@intel.com<br /> Release planned for Crow Pass<br /> Seeking new co-maintainer(s) to continue development towards production release |
> | valgrind<br /> pmemcheck | https://github.com/pmem/valgrind | Issue Reporting and Bug Fixes via GitHub and pmdk_support@intel.com<br /> Release planned for Crow Pass<br /> Seeking new co-maintainer(s) to continue development towards production release |
> ```

Is it still valid?

**Review by grom72 — COMMENTED** [review]
@wlemkows  Why?

> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/329#-:-NNqNZ4p3I7fTF6mZA5v:b-w9drbl)* status: 0 of 1 files reviewed, 3 unresolved discussions (waiting on @lukaszstolarczuk and @szadam)

**Review by grom72 — COMMENTED** [review]
I mixed revision sorry. Proper version is `No further support or maintenance planned``

> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/329#-:-NNqPtw6EpjxxxL39PiO:bl3do8f)* status: 0 of 1 files reviewed, 3 unresolved discussions (waiting on @lukaszstolarczuk and @szadam)

**Review by szadam — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/329)* status: 0 of 1 files reviewed, 3 unresolved discussions (waiting on @grom72 and @lukaszstolarczuk)

___
*[`content/blog/2022/update-pmdk-project-status.md` line 63 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/329#-NNqNAA-0Teg_pmY8Xof:-NNqSkaD16XweHZrP-VB:btpy3qq) ([raw file](https://github.com/pmem/pmem.github.io/blob/c5d2966fb68552c372d0c0cbc9beb113bb1c0e36/content/blog/2022/update-pmdk-project-status.md#L63)):*
<details><summary><i>Previously, grom72 (Tomasz Gromadzki) wrote…</i></summary><blockquote>

Is it still valid?
</blockquote></details>

I don't have any other details
___
*[`content/blog/2022/update-pmdk-project-status.md` line 64 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/329#-NNqNMFp6VWYPvwCnCBZ:-NNqSu5xAfnhZH_MScYY:badsjvs) ([raw file](https://github.com/pmem/pmem.github.io/blob/c5d2966fb68552c372d0c0cbc9beb113bb1c0e36/content/blog/2022/update-pmdk-project-status.md#L64)):*
<details><summary><i>Previously, grom72 (Tomasz Gromadzki) wrote…</i></summary><blockquote>

Is it still valid?
</blockquote></details>

 I don't have any other details

**Pull request merged**